common/models/schedule.json

```diff
@@ -1,11 +1,45 @@
 {
   "name": "Schedule",
-  "base": "Model",
+  "plural": "Schedules",
+  "base": "PersistedModel",
   "idInjection": true,
   "options": {
-    "validateUpsert": true
+    "validateUpsert": true,
+    "postgresql": {
+      "scheme": "picksapp",
+      "table": "schedule"
+    }
+  },
+  "properties": {
+    "timeLeft": {
+      "type": "string",
+      "required": true
+    },
+    "year": {
+      "type": "string",
+      "required": true
+    },
+    "week": {
+      "type": "string",
+      "required": true
+    },
+    "homeTeamName": {
```

Comment:
Any reason for the home team name?

Reply:
yes, for easy access to append other information gotten from another script/scrapper such as the weather information and rush yard etc.